src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -317,12 +314,10 @@ export class NotebookService extends Disposable implements INotebookService {
 	get onNotebookEditorAdd(): Event<INotebookEditor> {
 		return this._onNotebookEditorAdd.event;
 	}
+
 	get onNotebookEditorRemove(): Event<INotebookEditor> {
 		return this._onNotebookEditorRemove.event;
 	}
-	get onCellChanged(): Event<INotebookEditor> {
```

Comment:
This was deadcode with no reference to listeners being registered or the emitter behind this event being fired anywhere in the code.